selfdrive/test/process_replay/helpers.py

```diff
@@ -0,0 +1,25 @@
+import os
+import shutil
+import uuid
+
+from common.params import Params
+
+def setup_prefix():
```

Comment:
might be nice to turn the whole thing into a context manager

Reply:
good idea!